File: JS/URI/1043.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var line = lines.shift().split(" ");
var x = parseFloat(line[0]);
var y = parseFloat(line[1]);
var z = parseFloat(line[2]);


if(x+y > z && x+z > y && z+y > x)
    console.log("Perimetro = "+ (x+y+z).toFixed(1));

else
    console.log("Area = "+ (((x+y)*z)/2).toFixed(1));

File: Python/URI/1043.py
linha = input().split(' ')

x = float(linha[0])
y = float(linha[1])
z = float(linha[2])

if y+x > z and x+z > y and z+y > x:
    print("Perimetro = {:.1f}".format(x+y+z))
else:
    print("Area = {:.1f}".format((x+y)*z/2))